lib/Config/FetcherConfig.php

```diff
@@ -94,6 +94,32 @@ public function __construct(IConfig $config)
         return $this;
     }
 
+    /**
+     * Checks for available encoding options
+     * 
+     * @return String list of supported encoding types
+     */
+    public function checkEncoding()
+    {
+        $supportedEncoding = [];
+
+        $curl_version = curl_version();
```

Comment:
maybe add a comment that this is just checking the php curl extension to see which capabilities are enabled on the nextcloud instance